### ISSUE TITLE
Validation Set

### ISSUE THREAD

**Issue by pradhanpk** [pradhanpk/depot#2]
A plain random breakup could be suboptimal. See https://www.kaggle.com/c/home-depot-product-search-relevance/forums/t/18597/validation-set and https://www.kaggle.com/c/home-depot-product-search-relevance/forums/t/18479/local-cv

**Comment by pradhanpk** [comment]
The default sklearn `train_test_split` gives a test set whose score is 0.01-0.02 less than LB. Good enough for now.
